### ISSUE TITLE
Hello, what is the accuracy of your reproduction below?

### ISSUE THREAD

**Issue by yeyuxmf** [Andy-zhujunwen/danet-pytorch#2]
Hello, what is the accuracy of your reproduction below?
Whether it's training the original author's code or my own implementation, I only have about 70%. I don't know how much accuracy you can achieve?

**Comment by yeyuxmf** [comment]
@Andy-zhujunwen I train you to implement the code, and the results are as follows:
Nothing changed except batchsize. My batchsize = 2.
Validation:
[Epoch: 44, numImages:   500]
Acc:0.91330, Acc_class:0.69472, mIoU:0.59094, fwIoU:0.84864
Loss: 31.665
(mIoU)new pred =0.5909392207612304,old best pred=0.583372903387209

**Comment by Andy-zhujunwen** [comment]
> Hello, what is the accuracy of your reproduction below?
> Whether it's training the original author's code or my own implementation, I only have about 70%. I don't know how much accuracy you can achieve?


Frankly, this code does not reach the accuracy proposed by the danet paper.

**Comment by Evanchen1234** [comment]
pam模块很吃gpu显存，16GB显存瞬间爆满，你那里有这样的情况吗？

**Comment by yeyuxmf** [comment]
超级吃显存，10G的GPU没法玩， 也说明了这个模块的改进还有很大的空间

**Comment by xxyaoxxyao** [comment]
> 超级吃显存，10G的GPU没法玩， 也说明了这个模块的改进还有很大的空间

是在训练的时候吗

**Comment by pubinpubin** [comment]
> @Andy-zhujunwen I train you to implement the code, and the results are as follows:
> Nothing changed except batchsize. My batchsize = 2.
> Validation:
> [Epoch: 44, numImages: 500]
> Acc:0.91330, Acc_class:0.69472, mIoU:0.59094, fwIoU:0.84864
> Loss: 31.665
> (mIoU)new pred =0.5909392207612304,old best pred=0.583372903387209

同学关于训练和精度有问题向你请教下，可否留个qq或者联系方式，我的qq为：937140839

**Comment by wlj567** [comment]
您们好，请问这个复现需要什么环境，我这里无法运行

**Comment by wlj567** [comment]
@huang229 您好，方便加个联系方式，给一下您复现的DANet吗？如果可以，我的QQ是2795691748
